metrics/expvar/expvar.go

```diff
@@ -45,6 +49,8 @@ type gauge struct {
 	v *expvar.Float
 }
 
+func (g gauge) String() string { return fmt.Sprintf("%v", g.v) }
```

Comment:
String doesn't need to be in the definition for %v to return it. The String for Histogram needs String for Gauge and Counter to return all the values. It's a bit messy but is working well enough for me to make progress elsewhere. Need to discuss this when I see you later this week.